packages/graphql-algolia-integration/src/createAlgoliaSyncHandler.ts

```diff
@@ -64,18 +64,32 @@ const createAlgoliaSyncHandler = (config: LastRevAppConfig, graphQlUrl: string,
 
       const { errors: queryErrors, results } = await performAlgoliaQuery(apolloClient, config, contentStates);
 
+      if(!!queryErrors?.length) {
+        logger.error('[ERROR] performAlgoliaQuery, stopping execution', {
```

Comment:
maybe loop through the errors and log them, so that we can see the cause of the issues.

Reply:
Thanks @prceasar for the feedback. I'm already logging the error on line 57 of `performAlgoliaQuery.ts`